tests/rt.sh

```diff
@@ -7,7 +7,7 @@ die() { echo "$@" >&2; exit 1; }
 usage() {
   set +x
   echo
-  echo "Usage: $0 -c <model> | -f | -s | -l <file> | -m | -k | -r | -e | -h"
+  echo "Usage: $0 -c | -f | -s | -l <file> | -m | -k | -r | -e | -h"
   echo
   echo "  -c  create new baseline results for <model>"
```

Comment:
Remove `` for <model>`` from "echo" command.

Reply:
I did that, I think? Looks like at least.

Reply:
Ok, line 12 . Got it.